pkg/nimble/patches/0001-porting-nimble-src-nimble_port-riot-initializes-time.patch

```diff
@@ -0,0 +1,27 @@
+From 395209627ba495309098459173d40d490f680b8b Mon Sep 17 00:00:00 2001
+From: Francisco Molina <femolina@uc.cl>
+Date: Mon, 19 Apr 2021 15:47:49 +0200
+Subject: [PATCH] porting/nimble/src/nimble_port: riot initializes timers
+
+---
+ porting/nimble/src/nimble_port.c | 2 ++
+ 1 file changed, 2 insertions(+)
+
+diff --git a/porting/nimble/src/nimble_port.c b/porting/nimble/src/nimble_port.c
+index 484e3799..139179e1 100644
+--- a/porting/nimble/src/nimble_port.c
++++ b/porting/nimble/src/nimble_port.c
+@@ -45,8 +45,10 @@ nimble_port_init(void)
+ 
+ #if NIMBLE_CFG_CONTROLLER
+     ble_hci_ram_init();
++#ifndef MODULE_MYNEWT_CORE
```

Comment:
Just a thought, and I don't expect this to be fixed in this PR but maybe a follow up later: How about we exclude these two lines for all RIOT builds (e.g. -> `#ifndef RIOT_VERSION`) and call them if needed in the RIOT code in `nimble_riot_init()`(`pkg/nimble/contrib/nimble_riot.c`)?! This way we can get rid of the patch...